src/vcpkg/build.cpp

```diff
@@ -287,7 +345,7 @@ namespace vcpkg::Build
             {
                 msg::print(Color::warning, warnings);
             }
-            msg::print(Color::error, Build::create_error_message(result, spec));
+            msg::println_error(Build::create_error_message(result, spec));
```

Comment:
>Looking at Build::create_error_message, neither the function itself nor the corresponding message msgBuildingPackageFailed adds error: .

Ah, I see, *today* it adds the error prefix, but you changed it so that it does not.

>Again, just use find all references.

There's no "find all references" from the code review interface.